appstore/appstore/adapter.py

```diff
@@ -1,11 +1,55 @@
 from allauth.account.adapter import DefaultAccountAdapter
+from allauth.socialaccount.adapter import DefaultSocialAccountAdapter
+
+from django.conf import settings
 from django.forms import ValidationError
 
 
 class RestrictEmailAdapter(DefaultAccountAdapter):
-
-    def clean_email(self,email):
-        RestrictedList = ['Your restricted list goes here.']
+    def clean_email(self, email):
+        RestrictedList = ["Your restricted list goes here."]
         if email in RestrictedList:
-            raise ValidationError('You are restricted from registering. Please contact admin.')
```

Comment:
@n0mn0m what are we doing here? What is a restricted list?

Reply:
That already existed. I don't see where it is called though.